syntax = "proto3";

package image;

option go_package = "./image";

message ImageRequest {
 string prompt = 1;
}

message ImageResponse {
 string image_url = 1;
}

service ImageGenerator {
 rpc GetImageUrl (ImageRequest) returns (ImageResponse) {}
}
